pkg/client/unversioned/clientcmd/loader.go

```diff
@@ -137,9 +138,9 @@ func NewDefaultClientConfigLoadingRules() *ClientConfigLoadingRules {
 	envVarFiles := os.Getenv(RecommendedConfigPathEnvVar)
 	if len(envVarFiles) != 0 {
 		chain = append(chain, filepath.SplitList(envVarFiles)...)
-
 	} else {
 		chain = append(chain, RecommendedHomeFile)
+		chain = append(chain, loadConfigDir(RecommendedHomeFileDir)...)
```

Comment:
It would be more user-friendly if users could also specify directories. If file => use it. If directory => expand to its files and use them.